src/hw2/controllers/users.js

```diff
@@ -0,0 +1,112 @@
+import uuid from 'uuid/v1';
+
+import { schema, errorResponse } from '../validation/index';
+
+const users = {
+    1: {
+        id: '1',
+        login: 'ivan_hrushevich',
+        password: 'asdasd1!',
+        age: 17,
+        isDeleted: false
+    }
+};
+
+export const getFilter = (req, res) => {
```

Comment:
This route should be join with 'getUsers', so:

- GET /users should return all possible users
- GET /users?login=an should return all users with 'an' substring in login
- GET /users?limit=2 should return 2 first users (or less)
- GET /users?login=an&limit=2 should return 2 first users with 'an' substring in login (or less)